packages/wallets/package.json

```diff
@@ -28,12 +28,14 @@
         "react-router-dom": "^5.2.0",
         "react-transition-group": "4.4.2",
         "usehooks-ts": "^2.7.0",
-        "yup": "^0.32.11"
+        "yup": "^0.32.11",
+        "zxcvbn": "^4.4.2"
```

Comment:
This is 7 years since the last publish and has around 100 open [issues](https://github.com/dropbox/zxcvbn/issues). It also looks like it's over [300kb](https://bundlephobia.com/package/zxcvbn@4.4.2).

Not that untouched dependencies should be out of the question but are we really happy bringing this into our code?

Reply:
Yes I am aware of the status of the package. However, some of the warning messages are coming from the `zxcvbn` itself and it is used in Deriv app. That being said, will definitely find a more reliable package to use instead of this. Will improve this component as we go move further into v1 🙏🏼

Reply:
I don't see it being used anywhere in Deriv App right now. I can see we're using `@contentpass/zxcvbn` which is also questionable.

Reply:
They're essentially the same. The options out there for password validation are quite limited and `zxcvbn` happens to be one of the most use and reliable package compared to the rest. However like I said in the above this is not the final iteration we will make continuous improvements on the component.